src/CKEditorTrait.php

```diff
@@ -31,6 +32,57 @@
      */
     public $preset = 'standard';
 
+    /**
+     * Enable or disable kcfinder
+     * @link https://kcfinder.sunhater.com
+     * @var boolean
+     */
+    public $kcfinder = false;
```

Comment:
I think this should be a configuration array instead of a boolean value. Let me explain, if we have a `null` `kcFinderOptions` attribute, we would understand that there is no need to add a KCFinder or not. 

When not empty, then we could `create a new KCFinder instance` that will actually contain the `default` options + the method that will register the plugin. That way, we could have a cleaner `CKEditorTrait` (only one attribute), and move the logic of a specific finder `KcFinder`to another class. What do you think?

Reply:
I thinks about this before, But when the user need to kcfinder by default options, sure must set like this: `kcFinderOptions=>[]` , do you accept this way?!
And about separate logic, I review it.

Reply:
I think is fine as long as we documented it.